governance/xc_admin/packages/xc_admin_frontend/components/InstructionViews/WormholeInstructionView.tsx

```diff
@@ -408,12 +409,29 @@ export const WormholeInstructionView = ({
                 <CopyText text={'0x' + governanceAction.callAddress} />
               </div>
               <div>Value: {governanceAction.value.toString()}</div>
-              <div>
-                Call Data:{' '}
-                <CopyText
-                  text={'0x' + governanceAction.calldata.toString('hex')}
-                />
-              </div>
+              {(() => {
```

Comment:
this pattern of defining and running a function here seems a bit strange and inconsistent with how we handle these cases in the rest of the file.